config/storage.yml

```diff
@@ -7,12 +7,12 @@ local:
   root: "/media"
 
 # Use rails credentials:edit to set the AWS secrets (as aws:access_key_id|secret_access_key)
-# amazon:
-#   service: S3
-#   access_key_id: <%= Rails.application.credentials.dig(:aws, :access_key_id) %>
-#   secret_access_key: <%= Rails.application.credentials.dig(:aws, :secret_access_key) %>
-#   region: us-east-1
-#   bucket: your_own_bucket
+amazon:
+  service: S3
+  access_key_id: <%= Rails.application.credentials.dig(:aws, :access_key_id) %>
+  secret_access_key: <%= Rails.application.credentials.dig(:aws, :secret_access_key) %>
+  region: "us-east-2"
+  bucket: "terrastories-demo"
```

Comment:
Region and bucket should maybe be configurable through `ENV.fetch("ENV_VAR", "a_sensible_default")`. But they _aren't_ secrets, so they don't need to go through encrypted credentials.